src/server/StudioManagementSystem/Startup.cs

```diff
@@ -27,7 +53,34 @@ public void ConfigureServices(IServiceCollection services, IWebHostEnvironment e
 
         // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
         services.AddEndpointsApiExplorer();
-        services.AddSwaggerGen();
+        services.AddSwaggerGen(options =>
+        {
+            options.SwaggerDoc("v1", new OpenApiInfo { Title = "Studio Management Subject", Version = "v1" });
```

Comment:
nice touch! 🎉

_example change to the Swagger page_

![image](https://user-images.githubusercontent.com/26985949/235436045-dff14a93-cc9b-4c6b-96a1-2043b56a1974.png)